homeworks/test17/task6.py

```diff
@@ -0,0 +1,25 @@
+def work_with_file(path):
+    with open(path, 'r', encoding='utf-8') as file:
+        content_lines = file.readlines()
+    line_count = len(content_lines)
+    word_count = 0
+    for i in content_lines:
+        word_count += len(i.split())
+    letter_count = 0
+    for i in content_lines:
+        for word in i.split():
+            letter_count += len(word)
+    output_text = (
+        f'String count: {line_count}\n'
+        f'Word count: {word_count}\n'
+        f'Letter count: {letter_count}\n'
+    )
+    with open(path, 'a', encoding='utf-8') as file:
+        file.write(output_text)
+    print(output_text)
+
+
+work_with_file(
+    'C:\\Homework\\template-py-sdet-course-clone-hm4\\'
+    'homeworks\\test17\\task6.txt'
+)
```

Comment:
absolute path - not the best way to work with files.
Better to find it in the directory by name, if you know it https://realpython.com/working-with-files-in-python/#filename-pattern-matching